circleci/images/Makefile

```diff
@@ -0,0 +1,123 @@
+# config variable
+DOCKER_REPO=citus
+
+# auto generated variables
+ifeq ($(RELEASE),)
+	TAG_SUFFIX := -dev$(shell date +'%Y%m%d%H%M')
+else
+	TAG_SUFFIX :=
+endif
+
+# all postgres versions we test agains
+PG_VERSIONS = 11.10 12.5 13.1
+
+# citus version 8.x only supports 11, when bumping versions below we can create more images
+CITUS_UPGRADE_PG_VERSIONS = 11.10
+CITUS_UPGRADE_VERSIONS = v8.0.0 v8.1.0 v8.2.0 v8.3.0
+
+# code below creates targets for all postgres verions in PG_VERSIONS
+define make-image-targets
+# $1 = PG_VERSION
+# $2 = PG_MAJOR
+
+build-extbuilder-$1:
```

Comment:
Will try, last time I was experimenting with assigning a variable here it had strange side effects as it was only assigning the value on the second iteration. Kind of like an off by 1 error. Don't exactly understand how Makefiles work tbh ;)

Reply:
just some experimenting, when I add
```
PG_VERSION=$1
echo-$1:
	echo ${PG_VERSION}
```

to the define to see if I can get this to work this is my output:

```
DESKTOP-SKJOA71:images nilsdijk$ make echo-11.10
echo

DESKTOP-SKJOA71:images nilsdijk$ make echo-12.5
echo 11.10
11.10
DESKTOP-SKJOA71:images nilsdijk$ make echo-13.1
echo 12.5
12.5

Reply:
I see, This is not super important, I am just curious if we could give this directly as a named parameter, for example in your example if you do:

echo:
             echo ${PG_VERSION}

and when calling the target, we could do `make echo PG_VERSION=11.4`, do you think this would work?

Reply:
it currently generates the target by looping over the versions and calling the define. This allows for a lot less code and easier maintenance of the targets. That is also the cause of the positional arguments. They don't come from how make is called, but the internal call of the define.

I will leave this for now. It is slightly inconvenient, but it makes the file relatively short.